__tests__/utils/pages.js

```diff
@@ -0,0 +1,117 @@
+            ranking: [ "Rod Gardner & Leticia Gardner", "Derek Williams & Shelisa Williams", "Ricky Rotandi & Cesar Aldrete", "Anthony Smith & Bailey Smith", "Juan Villa & Shane Bilek", "Kishori Turner & Karishma Cordero", "Angie Butler & Danny Butler", "Michelle Clark & Sean Clark", "Yvonne Chavez & Melissa Main", "Amber Craven & Vinny Cagungun", "Sunny Pulver & Bizzy Smith", "Chris Foster & Mary Cardona-Foster", "Maya Mody & Rohan Mody"]
+        },
+        {
+            name: "Jim",
+            userId: "CAFA7731-EC3C-4B9D-8BC6-A199B6FB86A4",
+            ranking: [ "Ricky Rotandi & Cesar Aldrete", "Derek Williams & Shelisa Williams", "Rod Gardner & Leticia Gardner", "Michelle Clark & Sean Clark", "Juan Villa & Shane Bilek", "Chris Foster & Mary Cardona-Foster", "Sunny Pulver & Bizzy Smith", "Kishori Turner & Karishma Cordero", "Anthony Smith & Bailey Smith", "Yvonne Chavez & Melissa Main", "Amber Craven & Vinny Cagungun", "Angie Butler & Danny Butler", "Maya Mody & Rohan Mody" ]
+        },
+        {
+            name: "Rachel",
+            userId: "E3BA8CF1-0F66-4911-88D8-A9ECFEEB37A7",
+            ranking: ["Yvonne Chavez & Melissa Main", "Amber Craven & Vinny Cagungun", "Derek Williams & Shelisa Williams", "Ricky Rotandi & Cesar Aldrete", "Michelle Clark & Sean Clark", "Juan Villa & Shane Bilek", "Kishori Turner & Karishma Cordero", "Rod Gardner & Leticia Gardner", "Anthony Smith & Bailey Smith", "Sunny Pulver & Bizzy Smith", "Angie Butler & Danny Butler", "Chris Foster & Mary Cardona-Foster", "Maya Mody & Rohan Mody"]
+        },
+        {
+            name: "Sam",
+            userId: "CD9F9A71-FF9C-416A-8D0D-C268A13B021F",
+            ranking: ["Ricky Rotandi & Cesar Aldrete", "Rod Gardner & Leticia Gardner", "Yvonne Chavez & Melissa Main", "Derek Williams & Shelisa Williams", "Juan Villa & Shane Bilek", "Sunny Pulver & Bizzy Smith", "Amber Craven & Vinny Cagungun", "Angie Butler & Danny Butler", "Kishori Turner & Karishma Cordero", "Anthony Smith & Bailey Smith", "Michelle Clark & Sean Clark", "Chris Foster & Mary Cardona-Foster", "Maya Mody & Rohan Mody"],
+            handicap: -80
+        }
+    ];
+    return { CONTESTANT_LEAGUE_DATA };
+}
+
+const dataObject = {
+    BigBrother_26: {leagueConfig: BigBrotherConfigData()},
+    AmazingRace_36: {leagueConfig: AmazingRaceConfigData(), leagueData: AmazingRaceLeagueData()}
+}
+
+jest.mock("../../app/utils/pages", () => {
+    return {
+      ...jest.requireActual("../../app/utils/pages"),
+      getPathsToMap: jest.fn().mockImplementation(()=> { return Object.keys(dataObject)}),
+      getConfigurationPage: jest.fn().mockImplementation(filename => {
+        return dataObject[filename].leagueConfig;
+      }),
+      checkForSubpages: jest.fn().mockImplementation(filename => {
+        return dataObject[filename].hasOwnProperty('leagueData');
+      })
+    };
+});
+
+describe("pages getPages", () =>  {
+    it("should return appropriate data based on folder structure", () => {
```

Comment:
nit: I think a better name might be something... based on league status, and maybe something about order
```suggestion
    it("should return appropriate data order based on league status", () => {
```